01-exercise-easy/02-loops/loops_Dev.py

```diff
@@ -1,4 +1,4 @@
 if __name__ == '__main__':
     n = int(input())
-    
-    # Desenvolva a partir daqui
\ No newline at end of file
+    for i in range(0,n):
```

Comment:
@guilhermedoebber  o uso do `range(0, n)` é redundante, porque o valor inicial 0 é o padrão para o `range` então não é necessário adicionar ele. 

Pode ser feito desta maneira:
```suggestion
    for i in range(n):
```